.buildkite/pipeline.yml

```
@@ -10,3 +10,10 @@ steps:
     label: "Tests"
     agents:
       queue: "iOS"
+  - wait
+  - command: ".buildkite/build-static.sh"
+    label: ":hammer: Build Static"
+    artifact_paths:
+        - "MUXSDKStats.xcframework.zip"
```

We should name the artifacts differently to avoid confusion.
```suggestion
        - "MUXSDKStats-static.xcframework.zip"
```